Source/WebCore/Modules/webtransport/WebTransport.h

```diff
@@ -119,12 +121,12 @@ class WebTransport : public WebTransportSessionClient, public ActiveDOMObject {
     };
     State m_state { State::Connecting };
 
-    using PromiseAndWrapper = std::pair<Ref<DOMPromise>, Ref<DeferredPromise>>;
-    PromiseAndWrapper m_ready;
+    using PromiseAndWrapper = const std::pair<const Ref<DOMPromise>, const Ref<DeferredPromise>>;
```

Comment:
I think the first const should be removed.  All the members are const PromiseAndWrapper.